### PR TITLE
Unify "security profile" terminology to "signing certificate"

### PR DESCRIPTION
**Pull request by devinshoemaker**

### CODE REVIEW

**Comment by kevinports** [comment]
@devinshoemaker still seeing alot of references to 'security profile' when I do a global search in VScode 

For example these pages:
https://github.com/ionic-team/ionic-docs/blob/signing-credentials/src/pages/appflow/destinations/builds.md
https://github.com/ionic-team/ionic-docs/blob/signing-credentials/src/pages/appflow/destinations/manual.md
https://github.com/ionic-team/ionic-docs/blob/signing-credentials/src/pages/appflow/package/cli.md

**Comment by devinshoemaker** [comment]
@kevinports weird, not sure how I missed all that on my first pass. Will get those updated.

File: src/pages/appflow/quickstart/package.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ nextUrl: '/docs/appflow/quickstart/automation'
 Next we'll build an Android `Debug` binary using the [Ionic Package](/docs/appflow/package/intro) service.
 <blockquote>
 <b>Note:</b> If you'd like to build an Android <b>Release</b> binary or <b>any</b> iOS binary, you'll need to
-<a href="/docs/appflow/package/credentials">generate signing credentials</a>
-and <a href="/docs/appflow/package/adding-credentials">add them to a security profile</a> in the Appflow Dashboard.
+<a href="/docs/appflow/package/credentials">generate signing certificates</a>
+and <a href="/docs/appflow/package/adding-credentials">add upload them the Appflow dashboard.</a>
```
Comment (ltm) [inline src/pages/appflow/quickstart/package.md:14]:
```suggestion
and <a href="/docs/appflow/package/adding-credentials">upload them to the Appflow dashboard.</a> 
```

Comment (devinshoemaker) [reply src/pages/appflow/quickstart/package.md:14]: Good catch.

**Pull request merged**